extendr-api/src/wrapper/altrep.rs

```diff
@@ -1005,6 +1019,39 @@ impl Altrep {
         })
     }
 
+    #[cfg(use_r_altlist)]
+    pub fn make_altlist_class<StateType: AltrepImpl + AltListImpl + 'static>(
+        name: &str,
+        base: &str,
+    ) -> Robj {
+        #![allow(non_snake_case)]
+
+        single_threaded(|| unsafe {
+            let class = Altrep::altrep_class::<StateType>(Rtype::List, name, base);
+            let class_ptr = R_altrep_class_t { ptr: class.get() };
+
+            unsafe extern "C" fn altlist_Elt<StateType: AltListImpl + 'static>(
```

Comment:
Any reason to spell it `altlist_Elt` and not `altlist_elt`?

Reply:
Following convention of other altrep implementations which I think follows the convetion in the C file
https://github.com/wch/r-source/blob/49de65f04d675e485e783685e9786139fcb3b4c7/src/main/altrep.c#L202C34-L202C34